extensions/clusters/clusters.go

```diff
@@ -305,6 +306,10 @@ func NewRKE1ClusterConfig(clusterName string, client *rancher.Client, clustersCo
 		newConfig.DefaultPodSecurityAdmissionConfigurationTemplateName = clustersConfig.PSACT
 	}
 
+	if clustersConfig.AgentEnvVars != nil {
+		newConfig.AgentEnvVars = clustersConfig.AgentEnvVarsRKE1
+	}
+
```

Comment:
this is already set on line 270, do we need the if != nil logic here?

Reply:
it could be nil right Michael?